From 370510e24ddbf539392ebb6a1e43280965fcb19b Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Tue, 31 Mar 2009 18:47:32 +0530
Subject: [PATCH 10/69] DSS2: Small VRFB context allocation bug fixed

This is minor bug while requesting and mapping memory for
VRFB space.

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 drivers/video/omap2/omapfb/omapfb-main.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/video/omap2/omapfb/omapfb-main.c b/drivers/video/omap2/omapfb/omapfb-main.c
index 852abe5..44febef 100644
--- a/drivers/video/omap2/omapfb/omapfb-main.c
+++ b/drivers/video/omap2/omapfb/omapfb-main.c
@@ -1193,6 +1193,7 @@ static int omapfb_alloc_fbmem(struct fb_info *fbi, unsigned long size,
 
 		if(!va) {
 			printk(KERN_ERR "vrfb: ioremap failed\n");
+			omap_vrfb_release_ctx(&rg->vrfb);
 			return -ENOMEM;
 		}
 
-- 
1.6.2.4

